package.nls.json

```diff
@@ -12,5 +12,6 @@
     "commands.title.view.test.report": "View test report",
     "commands.title.add.project": "Add project to Liberty Dashboard",
     "commands.title.remove.project": "Remove project from Liberty Dashboard",
-    "commands.title.show.commands": "Show Liberty commands"
+    "commands.title.show.commands": "Show Liberty commands",
+    "views.welcome.message":" \n 1. If no projects are open in the VS Code Explorer, open or create a Liberty project using the File menu. \n 2. If there is one or more existing Maven or Gradle projects open in the VS Code Explorer, try one of the following: \n -Configure the Liberty build plugin in the build file of an existing Maven or Gradle project.\n - Add a server.xml file to an existing Maven or Gradle project at “src/main/liberty/config”.\n - Manually add an existing Maven or Gradle project to the Liberty Dashboard through the VS Code command palette action “Liberty: Add project to Liberty Dashboard”. "
```

Comment:
- If no projects are open in the VS Code Explorer, **open** or **create** a Liberty project using the File menu. (Can we add an action link for "open" and "create"?)
  - If there is one or more existing Maven or Gradle projects open in the VS Code Explorer, try one of the following:
    - Configure the Liberty build plugin in the build file of an existing Maven or Gradle project. [Click here for more Maven info](https://github.com/OpenLiberty/ci.maven/#configuration) or [click here for more Gradle info](https://github.com/OpenLiberty/ci.gradle#adding-the-plugin-to-the-build-script).
    - **Add a server.xml** file to an existing Maven or Gradle project at “src/main/liberty/config”. [Click here for more info](https://openliberty.io/docs/latest/reference/config/server-configuration-overview.html#server-xml). (Can we add an action link for “Add a server.xml file” where we generate an empty server.xml for the user?)
    - Manually add an existing Maven or Gradle project to the Liberty Dashboard through the VS Code command palette action “**Liberty: Add project to Liberty Dashboard**”. (Can we add an action link for “Liberty: Add project to Liberty Dashboard”?)
cc @aparnamichael

Reply:
(Can we add an action link for "open" and "create"?)
**I have added open project option. What is the "create" option , how should it work?**

(Can we add an action link for “Add a server.xml file” where we generate an empty server.xml for the user?)
**There is no build in command for create a file from welcome view. So i was not able to add this option.**

**All the remaining changes are done.**

<img width="469" alt="Screenshot 2023-11-24 at 7 41 05 PM" src="https://github.com/OpenLiberty/liberty-tools-vscode/assets/133862121/97817d76-4add-4e8a-b8ff-24d66df8f41c">

**Is this fine?**

Reply:
The screen shot looks OK, but can you please push your local changes to the branch on your fork so that I can try them out?

Reply:
@TrevCraw 
Push :**Don**e